packages/api/src/webhooks/cannon.ts

```diff
@@ -144,29 +144,21 @@ export default class WebhookCannon {
   }
 
   async handleWebhookQueue(data: ConsumeMessage) {
-    let message;
+    let trigger: messages.WebhookTrigger;
     try {
-      message = JSON.parse(data.content.toString());
-      console.log("webhookCannon: got message", message);
+      trigger = JSON.parse(data.content.toString());
+      console.log("webhookCannon: got trigger message", trigger);
     } catch (err) {
       console.log("webhookCannon: error parsing message", err);
       this.queue.ack(data);
       return;
     }
     try {
       // TODO Activate URL Verification
-      if (message.event && message.webhook && message.stream && message.user) {
```

Comment:
Why you've removed this check?

Reply:
Those are already checked in the fist line of the `_fireHook` function: https://github.com/livepeer/livepeer-com/blob/ebb97569b815880295da59fb530bb8aa88e550ff/packages/api/src/webhooks/cannon.ts#L218-L223

It also throws an error, which I think is better since it will be logged below. Just realized that the message will be unwillingly retried on that case, so I can fix that if you prefer. But IMO this is more of a sanity check that should never happen in production, so we don't want to silently ignore those cases anyway. 

I guess that we only really needed it before because of that `retry` bug that was actually sending the original `event` to the `*_single_url` queue, instead of the actual "webhook trigger" as I'm calling it here.

Reply:
> Just realized that the message will be unwillingly retried on that case, so I can fix that if you prefer

Yes, and that error will send message to retry loop. 
I've hit this problem when was developing and my old code sent message to the new one.

Everywhere we're getting data from external system we should check that is has format we expect. 
I think in this case we safely can just ignore such message.

Reply:
Fixed: https://github.com/livepeer/livepeer-com/pull/569/commits/340eb2dc808943f69470d4ae839b4e6247aceaef